frontend/src/components/modals/UpdateIssuerProfile.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <v-dialog v-model="dialog" max-width="500" persistent>
+    <v-card class="modal text-center">
+      <small class="modal-receiver">
+        {{ text }}
+      </small>
+      <div class="modal-contour">
+        <v-form class="pa-6">
+          <v-card-title class="headline justify-center">
+            {{ text }}
+          </v-card-title>
+          <ValidationObserver ref="validator">
+            <ValidationProvider
+              name="Issuer name"
+              rules="required|min:2|max:60"
+              v-slot="{ errors }"
+            >
+              <v-text-field
+                v-model="issuerProfile.name"
+                label="Issuer name"
+                placeholder="Please, enter the issuer name"
+                :error="Boolean(errors.length)"
+                :error-messages="errors"
+              />
+            </ValidationProvider>
+            <ValidationProvider rules="dimensions:500,500" name="Avatar">
+              <v-file-input
+                show-size
+                label="Select Image"
+                accept="image/*"
+                placeholder="Please, enter the issuer image"
+                @change="createBase64Image"
+              ></v-file-input>
+            </ValidationProvider>
+            <v-messages
+              v-if="error"
+              :value="[error]"
+              class="error--text mt-2"
+            />
+            <v-img
+              max-height="150"
+              max-width="250"
+              :src="issuerProfile.avatarUri"
+            ></v-img>
+            <v-card-actions class="justify-center my-5">
+              <v-btn
+                @click="closeModal"
+                color="info"
+                outlined
+                :disabled="processing"
+              >
+                Close
+              </v-btn>
+              <v-btn
+                @click="save"
+                color="info"
+                :disabled="processing"
+                :loading="processing"
+              >
+                Save
+              </v-btn>
+            </v-card-actions>
+          </ValidationObserver>
+        </v-form>
+      </div>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import ModalMixin from "../../mixins/ModalMixin";
+
+import { createNamespacedHelpers } from "vuex";
+const { mapActions: mapIssuerActions, mapState: mapIssuerState } =
+  createNamespacedHelpers("issuer");
+const { mapActions: mapUserActions } = createNamespacedHelpers("user");
+
+export default {
+  name: "UpdateIssuerProfile",
+  mixins: [ModalMixin],
+  data() {
+    return {
+      text: "Update Issuer Profile",
+      processing: false,
+      issuerProfile: {
+        name: "",
```

Comment:
Are you able to populate it with the current name, it would help rather than having an empty field for an edit.
Similarly, display the current avatar.

Also, the state is not cleaned on closing the modal. If I edit an Issuer "A", change its name to "AA" and save. Then I edit another Issuer "B", the modal will display "AA" in the field!

Reply:
I have fixed this bug , name is populated and avatar is displayed.